ui/src/app/base/components/TagSelector/TagSelector.tsx

```diff
@@ -184,21 +187,28 @@ export const TagSelector = ({
   header,
   showSelectedTags = true,
   tags = [],
+  useExternalTags = false,
```

Comment:
Could this just be determined by `externalSelectedTags.length > 0`?

Reply:
No, because initially the `externalSelectedTags` array could be empty (which would result in setting both internal and external tag state). Though we could just check for a truthy value (with `[]` being truthy). So if the array exists then we use that state. I'll update that.